src/library/trackset/crate/cratefeaturehelper.cpp

```diff
@@ -87,19 +87,20 @@ CrateId CrateFeatureHelper::createEmptyCrate() {
 CrateId CrateFeatureHelper::duplicateCrate(const Crate& oldCrate) {
     const QString proposedCrateName =
             proposeNameForNewCrate(
-                    QString("%1 %2").arg(
-                            oldCrate.getName(), tr("copy" , "[noun]")));
+                    QString("%1 %2")
+                            .arg( oldCrate.getName(), tr("copy", "[noun]")));
```

Comment:
The second argument in tr is deprecated. Use "//:"
You may also use + for string builder concatenation.

Reply:
Fixed in d40ade3dd2 - I hope that's how you meant it, at least the tests still work ;)

Btw you can use Github change suggestions for such things, that way there is no ambiguity.